.github/workflows/ci-cd.yml

```
@@ -0,0 +1,258 @@
+      # Add your production deployment steps here
+
+  performance-test:
+    name: Performance Testing
+    runs-on: ubuntu-latest
+    needs: [build]
+    if: github.event_name == 'pull_request'
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Run Lighthouse CI
+        uses: treosh/lighthouse-ci-action@v10
```

actionlint (1.7.8)

the runner of "treosh/lighthouse-ci-action@v10" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)